src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.helpers.ts

```diff
@@ -174,16 +159,54 @@ export class DatasetFlowTableHelpers {
         return "";
     }
 
-    public static waitingBlockText(startCondition: MaybeNull<FlowStartCondition>, datasetKind: DatasetKind): string {
+    public static durationBlockText(node: FlowSummaryDataFragment): string {
+        const descriptionTypename = node.description.__typename;
```

Comment:
Overall, I don't think duration blocks depends on flow type anyhow. It only depends on the start condition

Reply:
The label or anything else should also clearly indicate if this is a LEFT TO WAIT time, or AWAITING FOR SINCE time. As we display 2 kinds of information depending on the status/condition, and should somehow visually distinguish it.

Reply:
This block is shown only if the flow status is WAITING